dome9/resource_dome9_service_account.go

```diff
@@ -0,0 +1,119 @@
+package dome9
+
+import (
+	"github.com/dome9/dome9-sdk-go/dome9/client"
+	"github.com/dome9/dome9-sdk-go/services/serviceaccounts"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+	"log"
+)
+
+func resourceServiceAccount() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceServiceAccountCreate,
+		Read:   resourceServiceAccountRead,
+		Update: resourceServiceAccountUpdate,
+		Delete: resourceServiceAccountDelete,
+		Importer: &schema.ResourceImporter{
+			State: schema.ImportStatePassthrough,
+		},
+		Schema: map[string]*schema.Schema{
+			"name": {
+				Type:     schema.TypeString,
+				Required: true,
+			},
+			"api_key_id": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"api_key_secret": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"role_ids": {
+				Type:     schema.TypeList,
```

Comment:
Is "role_ids" represent an ordered collection of items? If no not, change it to another type "TypeSet".